webpack.config.example.js

```diff
@@ -59,5 +60,6 @@ module.exports = {
             loader: 'transform',
             query: 'brfs'
         }]
-    }
+    },
+    devtool: 'cheap-source-map' // don't use eval-related options
```

Comment:
I found `cheap-source-map` to be a good tradeoff but of course any other non-eval setting will do.